server/server.go

```diff
@@ -764,11 +764,13 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		DeleteLockCommand:  deleteLockCommand,
 	}
 
-	// No Storage backend configured for now
-	wsMux := websocket.NewMultiplexor(
-		logger,
-		controllers.JobIDKeyGenerator{},
-		projectCmdOutputHandler,
+	wsMux := websocket.NewInstrumentedMultiplexor(
+		websocket.NewMultiplexor(
+			logger,
+			controllers.JobIDKeyGenerator{},
+			projectCmdOutputHandler,
+		),
+		statsScope.SubScope("api"),
```

Comment:
For my understanding, can you explain the choice behind naming subscope `api`?

Reply:
This scope is used by the jobs controller which is a user-facing API. So, I figured api would be a relevant sub-scope for this use case! Do you have any scoping suggestions?

Reply:
Nope, it was just a little vague to me and I wanted to make sure I understood! Seeing the overall hierarchy here helps:
https://github.com/lyft/atlantis/pull/188#discussion_r812297647